src/main/java/carRacing/model/RacingRecords.java

```diff
@@ -0,0 +1,31 @@
+package carRacing.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingRecords {
+
+    List<RacingCar> records = new ArrayList<>();
+
+    public void registerCar(List<String> carNames){
+        for (String name : carNames){
```

Comment:
미션 요구사항 중 하나입니다.
> 자바 코드 컨벤션을 지키면서 프로그래밍한다.
이 과정의 Code Style은 [intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따른다.
[intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따르려면 code formatting 단축키(Windows : Ctrl + Alt + L. Mac : ⌥ (Option) + ⌘ (Command) + L.)를 사용한다.

```suggestion
public class RacingRecords {

    List<RacingCar> records = new ArrayList<>();

    public void registerCar(List<String> carNames) {
        for (String name : carNames) {
```
다른 클래스도 확인해 보시고 컨벤션을 지켜주면 좋겠습니다.
혼자서 하는 개인 프로젝트에는 별게 아닐 수 있지만
동료들과 협업할 때 일관성있게 작업을 하기 위해서는 필수라고 생각합니다.